test/e2e/performanceprofile/functests/11_mixedcpus/mixedcpus.go

```diff
@@ -303,6 +305,72 @@ var _ = Describe("Mixedcpus", Ordered, func() {
 
 			})
 		})
+		When("Disabling mixedCpus in the performance profile", func() {
+			It("should results with deployment pods that used shared cpus to failing to be scheduled", func() {
```

Comment:
`should causes all pods that uses shared CPUs to failed`

Reply:
or even better `should causes all pods that uses shared CPUs to fail`